csrankings-s.csv

```diff
@@ -1978,6 +1979,7 @@ Steven K. Feiner,Columbia University,http://www.cs.columbia.edu/~feiner,PogsVkYA
 Steven Kelk,Maastricht University,http://skelk.sdf-eu.org,Eoays_KFyQsC
 Steven L. Lytinen,DePaul University,http://condor.depaul.edu/slytinen/papers/WS505SmithT.pdf,NOSCHOLARPAGE
 Steven L. Tanimoto,University of Washington,http://www.cs.washington.edu/people/faculty/tanimoto,NOSCHOLARPAGE
+Steven L. Salzberg,Johns Hopkins University,https://salzberg-lab.org/,sUVeH-4AAAAJ

```

Comment:
<img width="1235" alt="Screen Shot 2024-08-05 at 10 05 09 AM" src="https://github.com/user-attachments/assets/cae7d03c-4120-4eba-a579-f5012844493f">

From this page: https://www.cs.jhu.edu/faculty/